rackcity/views/asset_views.py

```diff
@@ -1086,7 +1086,7 @@ def get_asset_from_barcode(request):
     barcodes = decode(opencv_img)
     if len(barcodes) < 1:
         return JsonResponse(
-            {"warning_message": "WARNING: No barcode detected"},
+            {"failure_message": "WARNING: No barcode detected"},
```

Comment:
picky but we probably don't need to show warning if it's not a warning :D

Reply:
Will fix in next push

Reply:
think u missed this one!